Gemfile

```diff
@@ -22,7 +22,7 @@ group :test do
   gem 'rubocop-shopify', '~> 1.0.7', require: false
   gem 'rubocop-performance', require: false
 
-  platform :mri, :truffleruby do
-    gem 'liquid-c', github: 'Shopify/liquid-c', ref: 'master'
-  end
+  # platform :mri, :truffleruby do
```

Comment:
Omit liquid C

Reply:
Good callout.  I guess we should put a comment here.  It does seem like forking that gem is going to be pretty involved.  Would be nice if we can drop the namespace altogether down the line so we don't have to fork it.

Reply:
actually, this is just the Gemfile for localdev of the gem - `platform` does not use `liquid-c`, so we won't be losing any performance with this fork